static/templates/compose_banner/message_sent_banner.hbs

```diff
@@ -1,4 +1,7 @@
-<div class="compose-notifications-content">
-    {{banner_text} {{#if link_text}}<a href="{{above_composebox_narrow_url}}" class="{{classname}}" data-message-id="{{link_msg_id}}">{{link_text}}</a>{{/if}}
-    <button type="button" class="out-of-view-notification-close close">&times;</button>
+<div class="above_compose_banner compose_banner success {{classname}}">
+    <p>
+        {{banner_text}}
+        {{#if link_text}} <a class="above_compose_banner_action_link" href="{{above_composebox_narrow_url}}" data-message-id="{{link_msg_id}}">{{link_text}}</a>{{/if}}
+    </p>
```

Comment:
Can you comment on the purpose/impact of adding the `p` tags here? I imagine it adjusts margins/whitespace in some way?